README.md

```diff
@@ -2,31 +2,322 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Create an interface
+- Implement an interface
+- Use the debugger to demonstrate polymorphism and dynamic binding
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+An **Interface** in Java is defined as an abstract type
+used to specify the behavior of a class.
+Interfaces are used to achieve abstraction, specifying
+what a class must do and not how. 
 
-## Topic 1
+## Code-Along
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+Fork and clone this lesson.  The code includes classes `Main`,
+`Rectangle` and `Circle`.  You will create an interface named `Shape`
+and adapt the classes to implement the interface.
 
-## Topic 2
+## Motivation
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+Let's look at an example application that demonstrates why interfaces are useful.
+Assume we own a restaurant with several rectangular tables, and we
+need to repaint the table tops.  We need to figure out the total
+area and circumference to determine how much paint to buy.
+
+The `Rectangle` class has methods to calculate area and circumference:
```

Comment:
This threw me for a minute using the word "circumference" instead of "perimeter" but I see what you did here.

Reply:
Hmmm, not sure why I used circumference instead of perimeter.  I updated the lesson to use perimeter.

Reply:
Maybe because a circle, which is introduced later, has a circumference rather than perimeter is my guess.